superset-frontend/src/SqlLab/components/ResultSet.tsx

```diff
@@ -100,6 +100,22 @@ const MonospaceDiv = styled.div`
   white-space: pre-wrap;
 `;
 
+const ReturnedRows = styled.div`
+  margin-top: 48px;
+  margin-bottom: -32px;
```

Comment:
there's some weirdness going on with the position: fixed styling of the controls, which is causing these weird margins. Let's see if can remove the position:fixed on the controls. I don't think they are supposed to scroll at all, so we may need to look for a container that has a scroll attribute on the overflow and remove it.

Reply:
@eschutho been playing around with this a bit. I got rid of position fixed on the controls, and if do overflow:hidden on the overall container then it replicates the behavior that we want without there being negative margins (I kind of had a theory that you might not like negative margin :) )

What I am currently doing is looking at how to target the overall container.

Reply:
ok, I think I fixed all of it with the last commit! This was a good dive into the various components in the South Pane.

Reply:
Looks like: 
![Screen Shot 2021-02-23 at 12 22 58 PM](https://user-images.githubusercontent.com/48933336/108884558-bbd8db80-75d4-11eb-97bd-89911abe79ea.png)

Reply:
yeah, that looks great! also, correct on the negative margins. :)